plugins/backstage-plugin-opa-authz-react/src/hooks/useOpaAuthz/useOpaAuthz.ts

```diff
@@ -9,6 +9,10 @@ export type AsyncOpaAuthzResult = {
   error?: Error;
 };
 
+export type ManualOpaAuthzResult = AsyncOpaAuthzResult & {
+  triggerFetch: () => Promise<PolicyResult | undefined>;
```

Comment:
```suggestion
  evaluatePolicy: () => Promise<PolicyResult | undefined>;
```

Sorry think I missed this one 👍